modules/ROOT/pages/con_pipelines-concepts.adoc

```diff
@@ -1,17 +1,17 @@
 [id='pipelines-concepts_{context}']
 = Pipelines concepts
 
-The Custom Resource Definitions (CRDs) provided by OpenShift Pipelines include: `Task`,`Pipeline`,`PipelineRun`, `TaskRun`, and `PipelineResource`.
+OpenShift Pipelines provide the following set of standard CRDs that act as the building blocks from which you can assemble a CI/CD pipeline for your application:
 
-* A `Task` is the smallest configurable unit in a `Pipeline` and is essentially a function of inputs and outputs that inform the `Pipeline` build. It can run individually or as a part of a `Pipeline`. A `Pipeline` includes one or more `Tasks`, with a `Task` consisting of one or more steps. Steps are a series of commands that are sequentially executed by the `Task`.
+Task:: Task is the smallest configurable unit in a Pipeline. It is essentially a function of inputs and outputs that form the Pipeline build. It can run individually or as a part of a Pipeline. A Pipeline includes one or more Tasks, where each Task consists of one or more steps. Steps are a series of commands that are sequentially executed by the Task.
 
-* A `Pipeline` is a series of `Tasks` that are executed to build complex workflows that automate the build, deployment and delivery of applications. It is a collection of `PipelineResources`,`Parameters`, and one or more `Tasks`. A `Pipeline` interacts with the outside world, by using `PipelineResources`, which are given to `Tasks` as inputs and outputs.
+Pipeline:: Pipeline consists of a series of Tasks that are executed to construct complex workflows that automate the build, deployment, and delivery of applications. It is a collection of PipelineResources, Parameters, and one or more Tasks. A Pipeline interacts with the outside world, by using PipelineResources, which are added to Tasks as inputs and outputs.
 
-* A `PipelineRun` is used to run and manage the execution of the `Pipeline`. `PipelineRun` initiates the creation of a `TaskRun` for each `Task` being executed in the `Pipeline`.
+PipelineRun:: PipelineRun is the running instance of a Pipeline. PipelineRun initiates a Pipeline and manages the creation of a _TaskRun_ for each Task being executed in the Pipeline.
 
-* `TaskRun` is auto-created by a `PipelineRun` for each `Task` in a `Pipeline` and  results from running an instance of a `Task`. It can also be manually created if a `Task` runs outside of a `Pipeline`.
+TaskRun:: is auto-created by a PipelineRun for each Task in a Pipeline. It is a result of running an instance of a Task in a Pipeline. It can also be manually created if a Task runs outside of a Pipeline.
```

Comment:
- Each of the definitions start with the term, to keep it consistent suggest TaskRun is auto...
- Same comment for PipelineResource.
or 

- If you wish to drop the term from all definitions we might want to construct all definitions in a different manner. For example: 

Task:: The smallest configurable unit in a Pipeline.
Pipeline:: A series of Tasks that are
TaskRun:: A result of running... It is auto-created...

- Also can you please help me understand why we dropped the article in all places? Seems a bit abrupt to me.

Reply:
Which Article?

Reply:
See the diff, the original had A Task, A pipeline etc.

Reply:
The Article was dropped because the opening statement of the paragraph was changed. See the opening statement "OpenShift Pipelines provide..."

Reply:
@Preeticp : 
__________________________________
Each of the definitions start with the term, to keep it consistent suggest TaskRun is auto...

Same comment for PipelineResource.
__________________________

have made it consistent with others.